libraries/ui-library/src/components/six-input/six-input.tsx

```diff
@@ -161,28 +161,28 @@ export class SixInput {
 
   @Watch('value')
   handleValueChange() {
-    if (!this.input) {
-      return;
+    this.value = this.getValue();
+    if (this.nativeInput && this.nativeInput.value !== this.value) {
+      this.nativeInput.value = this.value;
+      this.invalid = !this.nativeInput.checkValidity();
     }
-    this.input.value = this.value;
-    this.invalid = !this.input.checkValidity();
     this.sixValueChange.emit();
```

Comment:
Is sixValueChange emitted intentionally even if this.nativeInput.value has not changed and the input is not validated again. The behaviour of `setRangeText()` & `handleChange()` seem to be a bit different from the behaviour of `handleValueChange()` too.

Reply:
It was there before but looking at it again, it doesn't seem to make much sense to emit a special change event at all, if the value changes. I suggest to remove the `six-input-value-change` event all together.

Reply:
Indeed the idea is that we replicate the native element as much as possible. For this we also have a small test example on [docu page](https://six-group.github.io/six-webcomponents/components/six-input.html#events-discalimer ).

Since the native input element doesn't have such an event I would thus also vote to remove it. I don't really recall why I introduced it at the time. Maybe, because I want to have a more general event so I don't have to listen to `six-input-input` and `six-input-change` separately. However, I fully agree that we shouldn't introduce new conventions and follow the design of the native elements.